java/cli/src/main/java/org/ray/cli/CommandSubmit.java

```diff
@@ -0,0 +1,32 @@
+package org.ray.cli;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+/**
+ * Arguments for command submit
+ */
+@Parameters(separators = "= ", commandDescription = "submit a job to ray cluster")
+public class CommandSubmit {
+
+  // required
```

Comment:
It seems that we don't check if all required options are provided. or am I missing something?
btw, is the jcommander able to automatically validate the options?

Reply:
good suggestion. yes it does have one:)